marklogic-data-hub-central/ui/src/components/entities/matching/list-modal/list-modal.test.tsx

```diff
@@ -48,13 +50,62 @@ describe("Test input validation values to ignore", () => {
       updateListValues={() => {}}
       listValues={["One"]}
     />);
-    expect(screen.queryAllByText("Duplicated values is not allowed")).toHaveLength(0);
     const inputListValues =screen.getByPlaceholderText("Enter values to remove");
     fireEvent.change(inputListValues, {target: {value: "abcd"}});
     fireEvent.click(screen.getByRole("option"));
     fireEvent.change(inputListValues, {target: {value: "abcd"}});
     fireEvent.click(screen.getByRole("option"));
-    expect(screen.queryAllByText("Duplicated values is not allowed")).toHaveLength(1);
+    expect(screen.getByText(/already exists in this list/)).toBeInTheDocument();
+  });
+
+  it("Don't allow special characters on list of values", async () => {
+    global.document.createRange = () => ({
+      setStart: () => {},
+      setEnd: () => {},
+      commonAncestorContainer: {
+        nodeName: "BODY",
+        ownerDocument: document,
+      },
+    });
+    render(<ListModal
+      isVisible={true}
+      toggleModal={() => {}}
+      action={"A"}
+      confirmAction={() => {}}
+      checkIfExistInList={() => { return false; }}
+      updateListValues={() => {}}
+      listValues={["One"]}
+    />);
+    expect(screen.queryAllByText(errorText)).toHaveLength(0);
+    const inputListValues =screen.getByPlaceholderText("Enter values to remove");
+    fireEvent.change(inputListValues, {target: {value: "ab$cd"}});
+    fireEvent.click(screen.getByRole("option"));
+    expect(screen.queryAllByText(errorText)).toHaveLength(1);
+  });
+
+  it("on list of values doesn't contain spaces", async () => {
+    global.document.createRange = () => ({
+      setStart: () => {},
+      setEnd: () => {},
+      commonAncestorContainer: {
+        nodeName: "BODY",
+        ownerDocument: document,
+      },
+    });
+    render(<ListModal
+      isVisible={true}
+      toggleModal={() => {}}
+      action={"A"}
+      confirmAction={() => {}}
+      checkIfExistInList={() => { return false; }}
+      updateListValues={() => {}}
+      listValues={["One"]}
+    />);
+    expect(screen.queryAllByText(errorText)).toHaveLength(0);
+    const inputListValues =screen.getByPlaceholderText("Enter values to remove");
+    fireEvent.change(inputListValues, {target: {value: "ab cd"}});
+    fireEvent.click(screen.getByRole("option"));
+    expect(screen.queryAllByText(errorText)).toHaveLength(1);
```

Comment:
If possible, can we also add a check here to verify the latest duplicate values error?